core/admin/admin-notices.php

```diff
@@ -376,10 +377,49 @@ private function notice_experiment_promotion() {
 		return true;
 	}
 
-	public function print_admin_notice( array $options ) {
+	private function notice_design_not_appearing() {
+		$notice_id        = 'design_not_appearing';
+		$installs_history = get_option( 'elementor_install_history', [] );
+		$is_first_install = 1 === count( $installs_history );
+
+		if ( $is_first_install || ! current_user_can( 'update_plugins' ) || User::is_user_notice_viewed( $notice_id ) ) {
+			return false;
+		}
+
+		$options = [
```

Comment:
If it's used only inside the condition ($is_version_changed) maybe move it under the condition?